descartes_moveit/include/descartes_moveit/ikfast_moveit_state_adapter.h

```diff
@@ -31,7 +31,8 @@ class IkFastMoveitStateAdapter : public descartes_moveit::MoveitStateAdapter
   }
 
   virtual bool initialize(const std::string& robot_description, const std::string& group_name,
-                          const std::string& world_frame, const std::string& tcp_frame);
```

Comment:
I'm pretty sure that we will no longer need the IkFastMoveitStateAdapter once the IKFast templates are updated https://github.com/ros-planning/moveit/pull/1014#issuecomment-410066739